Server/src/index.js

```
@@ -0,0 +1,18 @@
+'use strict';
+import 'dotenv/config';
+import { createServer } from "./createServer.js";
+
+
+const PORT = process.env.PORT || 3005
+async function start() {
+  try {
+    const  {httpServer}=createServer()
+    httpServer.listen(PORT, () => {
+      console.log(`Server running at http://localhost:${PORT}/`)
+    })
+  } catch{
+console.log('server error start')
```

This `catch` block is a good idea for handling startup errors, but it currently swallows the error details. It's better to catch the specific error object (e.g., `catch (error)`) and log it (`console.error('Server failed to start:', error);`). This will make debugging much easier if something goes wrong during initialization.